src/bp/ui-admin/src/Pages/Server/Languages/api.ts

```diff
@@ -0,0 +1,13 @@
+import Axios, { AxiosInstance } from 'axios'
+
+import { LanguageSource } from './typings'
+
+export const getLanguageSourceClient = (source: LanguageSource): AxiosInstance => {
```

Comment:
We already have api.js which has the client for the API, licensing server & stripe. I suggest adding the language client there also, so we only have one source of truth

Reply:
I'm not sure I like the idea of having all api clients in a single file ... it makes everything depend on it ..